src/API/Instrumentation/Handler.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Instrumentation;
+
+use OpenTelemetry\API\Trace\Span;
+use OpenTelemetry\API\Trace\SpanKind;
+use OpenTelemetry\API\Trace\StatusCode;
+use OpenTelemetry\Context\Context;
+use Throwable;
+
+class Handler
```

Comment:
Maybe the name of this class could reflect that this is the handler specifically for creating spans? It is the most common purpose why functions are instrumented, but would still be nice for it to be explicit.

Reply:
Latter seems to make more sense to me.

Reply:
renamed to `WithSpanHandler`